deploy/runtools/runtime_config.py

```diff
@@ -211,7 +234,7 @@ def array_to_lognames(values: Sequence[Optional[Any]], prefix: str) -> List[str]
     def get_kill_simulation_command(self) -> str:
         driver = self.get_local_driver_binaryname()
         # Note that pkill only works for names <=15 characters
-        return """sudo pkill -SIGKILL {driver}""".format(driver=driver[:15])
+        return """pkill -SIGKILL {driver}""".format(driver=driver[:15])
```

Comment:
unless you've added sudo elsewhere, this won't work for drivers started with sudo

Reply:
`sudo` was added in a higher level: https://github.com/firesim/firesim/blob/a665e6be796b6c03ce72a096345e8095347b35cf/deploy/runtools/run_farm_deploy_managers.py#L218-L240. So the `sudo` will still exist on `sudo`-enabled systems and will be removed on non-`sudo` systems.